### DISCUSSION TITLE
macports update

### DISCUSSION THREAD

**Discussion by ihhub** [ihhub/fheroes2#3986] [General]
macports is right now at 0.9.4 version. Could we please bump it to 0.9.6?

**Comment by oleg-derevenetz** [comment]
https://github.com/macports/macports-ports/pull/11808

Maybe I'll not support versions earlier than 10.14 (IIRC) because they want special tweaks.

**Reply by ihhub** [reply]
I think it's fine. We don't need to support everything at this point.
